src/Tools/dotnet-dump/Analyzer.cs

```diff
@@ -54,7 +55,7 @@ public Task<int> Analyze(FileInfo dump_path, string[] command)
                 using DataTarget dataTarget = DataTarget.LoadDump(dump_path.FullName);
 
                 OSPlatform targetPlatform = dataTarget.DataReader.TargetPlatform;
-                if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX)) {
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX) || dataTarget.DataReader.EnumerateModules().Any((module) => Path.GetExtension(module.FileName) == ".dylib")) {
```

Comment:
clrmd doesn't know about the ELF dumps we generate on MacOS.  This is a hack that allows them to be loaded on Windows for mostly testing/debugging the low level parts of the infrastructure. Since there isn't a cross-OS DAC for MacOS the clrmd or SOS commands won't work but it helps me debug the rest.